src/ring_theory/ideals.lean

```diff
@@ -216,6 +216,8 @@ instance (I : ideal α) : comm_ring I.quotient :=
 /-- `ideal.quotient.mk` as a `ring_hom` -/
 def mk_hom (I : ideal α) : α →+* I.quotient := ⟨mk I, rfl, λ _ _, rfl, rfl, λ _ _, rfl⟩
 
+lemma mk_eq_mk_hom (I : ideal α) (x : α) : ideal.quotient.mk I x = ideal.quotient.mk_hom I x := rfl
```

Comment:
Which form is `simp` normal? Usually I added a `simp` lemma `coe_mk_hom`, possibly with `mk_hom_apply`. Then you can rewrite right to left if needed.

Reply:
I don't think we have settled on a simp normal form. But since `mk` is not a `coe`, I'm hoping that steadily everything will become `mk_hom`. (And maybe `mk` will just be a `ring_hom` to start with.) But I'm not making this simp, because that would probably break a lot downstream.